src/frontend/apps/desk/src/app/Language/LanguagePicker.tsx

```diff
@@ -31,49 +33,42 @@ const SelectStyled = styled(Select)<{ $isSmall?: boolean }>`
   }
 `;
 
-const optionsPicker = [
-  {
-    value: 'FR',
-    label: 'FR',
-    render: () => (
-      <Box
-        className="c_select__render"
-        $direction="row"
-        $gap="0.7rem"
-        $align="center"
-      >
-        <Image priority src={IconLanguage} alt="Language Icon" />
-        <Text>FR</Text>
-      </Box>
-    ),
-  },
-  {
-    value: 'EN',
-    label: 'EN',
-    render: () => (
-      <Box
-        className="c_select__render"
-        $direction="row"
-        $gap="0.7rem"
-        $align="center"
-      >
-        <Image priority src={IconLanguage} alt="Language Icon" />
-        <Text>EN</Text>
-      </Box>
-    ),
-  },
-];
-
 export const LanguagePicker = () => {
+  const { t, i18n } = useTranslation();
+  const { preload: languages } = i18n.options;
```

Comment:
I faced some issue while loading translation, and all languages were still accessible in the dropdown. By default, if translation are not loaded properly, we would prefer have a check, preventing the user selecting a language which has not been loaded.

Reply:
We have test to prevent that, you cannot merge a branch when the translations are not made.
https://github.com/numerique-gouv/people/blob/94a68fb68301f1ba674c6c919c8e1533831a8d9d/src/frontend/packages/i18n/__tests__/translations.test.ts#L4

Reply:
I am not sure your answer, matches my question. I am concerned about the Language selector showing all languages, even if translations have not been pulled from Crowdin. Let's discuss it orally.